src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep.java

```diff
@@ -319,6 +319,10 @@ private void validateGeneralSettings(boolean isAppScan360, Map<String, String> p
             m_scanner = ScannerFactory.getScanner(Scanner.SOFTWARE_COMPOSITION_ANALYZER, properties.get(CoreConstants.TARGET));
             properties.put(CoreConstants.SCANNER_TYPE, CoreConstants.SOFTWARE_COMPOSITION_ANALYZER);
         }
+
+        if(properties.containsKey(CoreConstants.SCAN_ID) && !ServiceUtil.isScanId(properties.get(CoreConstants.SCAN_ID), m_authProvider, properties)) {
```

Comment:
Is this intended for the backend scan ID check? targeting the Pipeline script?

Reply:
Yes Kripa, I have changed the "isScanId()" method a little bit. Instead of passing the complete properties map we should pass the required strings only i.e. 'AppId' & 'ScanType'.